tests/setup_tests.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh -xe
+
+# This script starts docker and systemd (if el7)
+
+# Version of CentOS/RHEL
+el_version=$1
+
+ # Run tests in Container
+if [ "$el_version" = "6" ]; then
+
+sudo docker run --rm=true -v `pwd`:/htcondor-ce:rw centos:centos${OS_VERSION} /bin/bash -c "bash -xe /htcondor-ce/tests/test_inside_docker.sh ${OS_VERSION}"
+
+elif [ "$el_version" = "7" ]; then
+
+docker run --privileged -d -ti -e "container=docker"  -v /sys/fs/cgroup:/sys/fs/cgroup -v `pwd`:/htcondor-ce:rw  centos:centos${OS_VERSION}   /usr/sbin/init
+DOCKER_CONTAINER_ID=$(docker ps | grep centos | awk '{print $1}')
+docker logs $DOCKER_CONTAINER_ID
+docker exec -ti $DOCKER_CONTAINER_ID /bin/bash -c "bash -xe /htcondor-ce/tests/test_inside_docker.sh ${OS_VERSION};
```

Comment:
At the top, the argument is `#!/bin/bash -xe`, the `e` of which means it will it will exit at the first non-zero exit command, with that status.  So, if this script fails, the build will fill.